### ISSUE TITLE
sRGB color profile does not work on macOS (1.28.0)

### ISSUE THREAD

**Issue by atomiks** [atom/atom#17564]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: https://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

Selecting `Use sRGB color profile` has no effect on macOS 10.13.4.

### Steps to Reproduce

**Default**

<img src="https://i.imgur.com/7rbPgZi.png" height="400" />

**sRGB**

<img src="https://i.imgur.com/sJOo2Li.png" height="400" />

OS color profile:

<img src="https://i.gyazo.com/685192c3916b7dc8ed0361c9b57d17c5.png" height="400" />

**Expected behavior:** sRGB should have the same color profile as 1.27.0

**Actual behavior:** sRGB color profile doesn't work. Using the sRGB OS color profile works, but it enforces the profile everywhere which causes oversaturated imagery. I like it just in my editor for the vibrant syntax colors.

**Reproduces how often:** 100%

### Versions

1.28.0

### Additional Information

**Comment by thomasjo** [comment]
Just double checking since this isn't mentioned in the issue body; did you restart Atom after changing the setting? A prompt/dialog should be triggered by changing the setting, which has a restart button.

The setting is seemingly working for everyone else, so we'll have to figure out why it's not working on your system.

**Comment by atomiks** [comment]
Yes, I tried at least 5 times. I also restarted my computer.

I'll try a fresh install of Atom altogether

**Comment by rsese** [comment]
> I'll try a fresh install of Atom altogether

Not sure if you only reinstalled Atom, but did you also temporarily reset to factory defaults?

https://flight-manual.atom.io/hacking-atom/sections/debugging/#reset-to-factory-defaults

Then change the setting and restart again?

Also, are there any errors in devtools console?

https://flight-manual.atom.io/hacking-atom/sections/debugging/#check-for-errors-in-the-developer-tools

**Comment by atomiks** [comment]
I deleted the hidden `.atom` directory before reinstalling completely, and it seemed to have everything reset completely to default. There are no errors in the console.

**Comment by jasonrudolph** [comment]
> Expected behavior: sRGB should have the same color profile as 1.27.0

@atomiks: Can you share a few additional screenshots to help us understand the issue?

1. Screenshot of the colors you're seeing in Atom 1.27 on macOS 10.13.4
1. Screenshot of the colors you're seeing in Atom 1.28 on macOS 10.13.4 using Atom's default color profile setting
1. Screenshot of the colors you're seeing in Atom 1.28 on macOS 10.13.4 using Atom's sRGB color profile setting

As a bonus, if you can use the Digital Color Meter app to show us the exact RGB values in each screenshot (e.g., as seen in https://github.com/atom/atom/pull/17380#issuecomment-394015699), that would be extra helpful. :pray:

**Comment by atomiks** [comment]
The colors are the exact same using the sRGB setting and default color profile setting, as seen in the top two screenshots. I've changed the hex color slightly since, but here are the values:

After pasting the colors in Photoshop.

Default 
<img src="https://i.imgur.com/o8NCpwK.png" height="200" />

The red is: #f07da4
The actual red hex color in the syntax theme: #ff74a5 

sRGB
<img src="https://i.imgur.com/TYZ2sTa.png" height="200" />

Photoshop seems to use the sRGB color profile (or maybe Display P3?). I'm using a 2016 MBP which has the wide gamut display so I'm not sure if the colors can actually be seen on other screens via screenshots 🤔 .

Also, you can't see a difference between the images in Chrome because it's using the OS color profile setting. You need to download them and view them on a wide gamut display.

Basically there's no way to show you the differences easily... all I know is that 1.27.0 had the vibrant colors and 1.28.0 is desaturated

**Comment by atomiks** [comment]
The colors are the exact same using the sRGB setting and default color profile setting in 1.28.0, as seen in the top two screenshots. I've changed the hex color slightly since, but here are the values:

After pasting the colors in Photoshop.

1.28.0 (with sRGB setting) 
<img src="https://i.imgur.com/o8NCpwK.png" height="200" />

The red is: #f07da4
The actual red hex color in the syntax theme: #ff74a5 

Photoshop seems to use the sRGB color profile (or maybe Display P3?). I'm using a 2016 MBP which has the wide gamut display so I'm not sure if the colors can actually be seen on other screens via screenshots 🤔 .

1.27.0 vs 1.28.0

<img src="https://i.gyazo.com/862a5a2f1a51dc791f774e01eeec2b20.png" />

**Comment by matt3224** [comment]
I'm seeing this in Mojave with 1.31.0, changing to sRGB has no effect.
